src/commercetools/testing/predicates.py

```diff
@@ -394,8 +397,11 @@ def __repr__(self):
 
 
 class PredicateFilter:
-    def __init__(self, predicate):
+    operators = {">": operator.gt, "=": operator.eq, "is not": operator.is_not}
```

Comment:
For my understanding, shouldn't we have more operators here (f.e. support 'age < 42') ?

Reply:
Yup! I’ll add more indeed